src/components/pages/landingpage/career.tsx

```diff
@@ -30,7 +40,7 @@ export const Career = ({ positions }: CareerProps) => {
       <TeaserGrid>
         {positions.map((item) => (
           <Teaser key={item.id} title={item.name} linkTo={item.slug}>
-            {item.short}
+            {textEllipsis(item.short, 200)}
```

Comment:
We have some lengthy short descriptions so I decided to introduce a text ellipsis. I'm sure I could do this with CSS or even through graphql but this was the easiest way for now.